docs/framework/unmanaged-api/profiling/icorprofilerinfo8-interface.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "ICorProfilerInfo8 Interface"
+ms.date: "08/06/2019"
+author: "davmason"
+ms.author: "davmason"
+---
+# ICorProfilerInfo8 Interface
+
+A subclass of [ICorProfilerInfo7](../../../../docs/framework/unmanaged-api/profiling/icorprofilerinfo7-interface.md) that provides methods to query information about dynamic methods.
+
+## Methods  
+
+| Method|Description|  
+| ------------|-----------------|  
+|[IsFunctionDynamic Method](../../../../docs/framework/unmanaged-api/profiling/icorprofilerinfo8-isfunctiondynamic-method.md)| Determines if a function has associated metadata.|
```

Comment:
If the description of IsFunctionDynamic changes, this description should be changed too. (See my comment.)